samples/local_attestation/Makefile

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+.PHONY: all build clean run
```

Comment:
@aserdean - Shouldn't the order be clean build run?

Reply:
@anitagov the actual order does not matter, I just chose an alphabetical one 😄 .
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

This PR mainly is to fix the issue if someone is using CMake and creates a `build` directory while also trying to use the GNU makefiles. For that case we needed a phony target `build`. I added the rest for the theoretical performance gain.

Reply:
OK, great. Thanks for the interesting link.